s3/pkg/meta/types/object.go

```diff
@@ -120,11 +120,15 @@ func (o *Object) GetRowkey() (string, error) {
 }
 
 func (o *Object) GetValues() (values map[string]map[string][]byte, err error) {
-	var size bytes.Buffer
+	var size, tier bytes.Buffer
 	err = binary.Write(&size, binary.BigEndian, o.Size)
 	if err != nil {
 		return
 	}
+	err = binary.Write(&tier, binary.BigEndian, o.Tier)
```

Comment:
why do we replace Size with Tier here, sorry, DNU

Reply:
Hi, we use tier, not storage class internal, and we also have a map from internal tier to external storage class of a specific cloud.